src/PositionDropBoxData.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once 'src/Models/PlayerCardModel.php';
+require_once 'src/DatabaseConnection.php';
+
+function PositionData(PlayerCard $card)
+{
+    $db = DatabaseConnection();
+    $positionsData = new PlayerCardModel($db);
+    $positions = $positionsData->getPositionData();
```

Comment:
Instead of instantiating these within this function, you should setup the function so that it gets passed the model rather than creates it (dependency injection)